study.md

```diff
@@ -44,21 +44,23 @@ what do developers mean when they say "view-state"? How do front-end "routers"
 help manage view-state?
 
 ```md
-<!-- your answer here -->
+The "view" is an HTML fragment that has a specific set of functionality to the application.  It refers to everything the user sees.
 ```
```

Comment:
make sure to answer the whole question! This repo breaks down view vs. view state pretty well!
https://github.com/ga-wdi-boston/spa-study/blob/master/study.md